### PR TITLE
created updated for profiles pref

### PR DESCRIPTION
**Pull request by kaamil2**
Screenshot 2024-09-28 at 5 23 29 PM
Screenshot 2024-09-28 at 5 23 26 PM
# Description

[Link to Ticket](insert the link to your ticket inside the parenthesis here)

Please include a summary of the changes and the related issue. Please also
include relevant motivation, context, and images!

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. If they are unit
tests, provide the file name the tests are in. If they are not unit tests,
describe how you tested the change.

# Checklist

- [ ] I have performed a self-review of my code
- [ ] I have reached out to another developer to review my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by CamPlume1 — APPROVED** [review]
Looks really great

File: backend/internal/types/params.go [hunk 1]
```diff
@@ -3,9 +3,18 @@ package types
...
+	"github.com/google/uuid"
 )
 
 type Params struct {
 	Supabase config.Supabase
 	Store    storage.Storage
 }
+
+type UpdateProfilePrefencesRequest struct {
+	UserID            uuid.UUID `json:"user_id"`
```
Comment (CamPlume1) [inline backend/internal/types/params.go:15]: This should be in models not types

**Review by CamPlume1 — CHANGES_REQUESTED** [review]

File: backend/internal/types/params.go (hunk 1 above)
Comment (kaamil2) [reply backend/internal/types/params.go]: think this is good now

**Review by CamPlume1 — APPROVED** [review]

File: backend/internal/storage/storage.go [hunk 2]
```diff
@@ -30,6 +31,9 @@ type UserRating interface {
 	GetAllUserRatings(context.Context, uuid.UUID) ([]models.UserRating, error)
 }
 
+type Profiles interface {
+	UpdateProfilePrefences(context.Context, uuid.UUID, string, string, string, string) error
```
Comment (CamPlume1) [inline backend/internal/storage/storage.go:35]: This should be combined with the Profile interface, as they cover the same db table

**Review by CamPlume1 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**